src/main/java/com/_up/megastore/controllers/interfaces/IProductController.java

```diff
@@ -18,4 +19,11 @@ ProductResponse saveProduct(
             @RequestPart MultipartFile multipartFile
     );
 
+    @PatchMapping(value = "/{productId}", consumes = { MediaType.MULTIPART_FORM_DATA_VALUE, MediaType.APPLICATION_JSON_VALUE })
+    @ResponseStatus(HttpStatus.OK)
+    ProductResponse updateProduct(
+            @RequestPart @Valid UpdateProductRequest updateProductRequest,
```

Comment:
Esto debería ser un RequestBody mas que RequestPart

```suggestion
            @RequestBody @Valid UpdateProductRequest updateProductRequest,
```